bundles/org.smarthomej.binding.notificationsforfiretv/src/main/java/org/smarthomej/binding/notificationsforfiretv/internal/NotificationsForFireTVHandler.java

```diff
@@ -91,29 +93,40 @@ public boolean sendNotification(@Nullable String msg, @Nullable String filename,
             notificationsForFireTVConnection.addFormField(APP, config.title);
             notificationsForFireTVConnection.addFormField(FORCE, String.valueOf(config.force));
             notificationsForFireTVConnection.addFormField(INTERRUPT, String.valueOf(0));
-            // ADD FILES
+            // ADD FILE PARTS
             if (filename != null) {
                 File file = new File(filename);
                 if (!file.exists()) {
-                    throw new IOException("File doesn't exist: " + filename);
+                    logger.warn("File doesn't exist: {}", filename);
+
+                    return false;
                 }
                 notificationsForFireTVConnection.addFilePart(FILENAME, file);
             }
             if (filename2 != null) {
                 File file2 = new File(filename2);
                 if (!file2.exists()) {
-                    throw new IOException("File doesn't exist: " + filename2);
+                    logger.warn("File doesn't exist: {}", filename2);
+
+                    return false;
                 }
                 notificationsForFireTVConnection.addFilePart(FILENAME2, file2);
             }
             // POST FORM
             notificationsForFireTVConnection.finish();
 
+            // UPDATE STATUS
+            updateStatus(ThingStatus.ONLINE);
+
             return true;
-        } catch (Exception e) {
-            logger.warn("Unable to connect: {}", e.getMessage());
+        } catch (IOException e) {
+            logger.debug("Unable to send notification: {}", e.getMessage());
```

Comment:
```suggestion
            logger.debug("Unable to send notification: ", e);
```

Either log the full stack trace here (if it add useful information) or remove the logging. The change of the thing status is logged anyway, so updating the thing status already logs `e.getMessage()`

Reply:
done